assets/src/data/content/editors.ts

```diff
@@ -1,9 +1,7 @@
-import React from 'react';
-
 export type EditorDesc = {
   slug: string;
-  deliveryElement: string | React.FunctionComponent;
-  authoringElement: string | React.FunctionComponent;
+  deliveryElement: string;
```

Comment:
I couldn't find any places where this was actually using this as a react component, so this change seemed to make sense and simplified things a bit